docs/csharp/language-reference/statements/snippets/lock/Program.cs

```diff
@@ -3,7 +3,8 @@
 
 public class Account
 {
-    private readonly object balanceLock = new object();
+    // Use `object` in versions earlier than C# 13
+    private readonly System.Threading.Lock balanceLock = new();
     private decimal balance;
```

Comment:
If you accept these suggestions, you'd have to update the other places in the code where these fields are used.

```suggestion
    private readonly System.Threading.Lock _balanceLock = new();
    private decimal _balance;
```